scripts/migrations/6-state-stake.py

```diff
@@ -0,0 +1,22 @@
+import sys
+import os
+import json
+from collections import OrderedDict
+
+home = sys.argv[1]
+output_home = sys.argv[2]
+
+config = json.load(open(os.path.join(home, 'output.json')), object_pairs_hook=OrderedDict)
+
+assert config['protocol_version'] == 5
+
+config['protocol_version'] = 6
+config['runtime_config'].pop('storage_cost_byte_per_block')
+config['runtime_config'].pop('account_length_baseline_cost_per_block')
+config['runtime_config']['storage_amount_per_byte'] = 90949470177292823791
```

Comment:
Please add a comment for what's added in the version, by pointing to the NEP

Reply:
Linked and described.